gc/base/Heap.hpp

```diff
@@ -165,7 +165,7 @@ class MM_Heap : public MM_BaseVirtual
 	void initializeCommonGCStartData(MM_EnvironmentBase *env, struct MM_CommonGCStartData *data);
 	void initializeCommonGCEndData(MM_EnvironmentBase *env, struct MM_CommonGCEndData *data);
 
-	uintptr_t getActualSoftMxSize(MM_EnvironmentBase *env);
+	uintptr_t getActualSoftMxSize(MM_EnvironmentBase *env, uintptr_t memoryType = MEMORY_TYPE_OLD);
```

Comment:
I'd remove the default value. Not much reason to prefer OLD over NEW, and there are probably not that many spots that call this to justify code savings, while on the other side, it may potentially confuse reader without explicit argument.

Reply:
ok, leave it with default value, then